tasks/config-opsdir/task.sh

```diff
@@ -44,6 +45,38 @@ AZ_CONFIGURATION=$(cat <<-EOF
 }
 EOF
 )
+elif [ ! -z $AZ_2 ];then
+AZ_CONFIGURATION=$(cat <<-EOF
+{
+  "availability_zones": [
+    {
+      "name": "$AZ_1",
+      "cluster": "$AZ_1_CLUSTER_NAME",
+      "resource_pool": "$AZ_1_RP_NAME"
+    },
+    {
+      "name": "$AZ_2",
+      "cluster": "$AZ_2_CLUSTER_NAME",
+      "resource_pool": "$AZ_2_RP_NAME"
+    }
+  ]
+}
+EOF
+)
+else
+AZ_CONFIGURATION=$(cat <<-EOF
+{
+  "availability_zones": [
+    {
+      "name": "$AZ_1",
+      "cluster": "$AZ_1_CLUSTER_NAME",
+      "resource_pool": "$AZ_1_RP_NAME"
+    }
+  ]
+}
+EOF
+)
+fi
```

Comment:
@krishicks first off, thank you for the great work you've put into the pipelines to get them to this point. It's a needed and valuable addition to the product suite! I am a bit concerned though on the premise these are only developed to conform to the reference architecture. I can definitely appreciate the need to have an initial target. But as someone working at customer sites, the norm is not the reference architect. Enterprises always have nuances and various reasons as to why the canonical reference architecture doesn't work. Limiting the pipelines to just the reference architecture unfortunately decreases the ability to use the pipelines in real world situations. I would purpose taking a slightly different approach where the initial pipeline is created to deploy the ref arch and then let it evolve as real world needs arise. And let the pipeline grow organically based on those requirements, which could be manifested in the form of pull requests. For example, this pull request is one that would be needed at every Enterprise in which I've install PCF.